tests/e2e/pact/broker/publish.ts

```diff
@@ -3,13 +3,15 @@ import { Publisher } from "@pact-foundation/pact"
 
 async function publish(): Promise<Array<string>> { 
   const isLocal = process.env.PACT_PROVIDER_URL == "http://localhost:9000"
+  const isInt = process.env.APIGEE_ENVIRONMENT === "int"
+  const pactVersion = isInt ? `${process.env.PACT_VERSION}.int` : process.env.PACT_VERSION
```

Comment:
Why's this different in int?

Reply:
So that the ids generated for int are in a separate pact for sending to suppliers and referencing in the tracker and not overwritten by other environment deploys